packages/web-components/fast-components/src/card/README.md

```diff
@@ -1,4 +1,4 @@
 # fast-card
-The `fast-card` component is a visual container and design system provider. By default `fast-card` applies `neutralFillCard` to its background that is calculated from its parent design system provider. If a custom background color is desired the attribute `card-background-color` is available, this will reset the cards design system to that value. Cards are snapshots of content that are typically used in a group to present collections of related information.
+The `fast-card` component is a visual container and design system provider. By default `fast-card` applies `neutralFillCard` to its background that is calculated from its parent design system provider. Cards are snapshots of content that are typically used in a group to present collections of related information.
```

Comment:
```suggestion
The `fast-card` component is a visual container. By default `fast-card` applies `neutralFillLayerRecipe` to its background that is calculated from its parent's fill color. Cards are snapshots of content that are typically used in a group to present collections of related information.
```

Reply:
Looks great.

Reply:
I'm going to commit this and rebuild so we can get this in.